recipe-server/client/control/components/action_fields/OptOutStudyFields.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes as pt } from 'react';
+
+import { ControlField } from 'control/components/Fields';
+import ActionFields from 'control/components/action_fields/ActionFields';
+
+/**
+ * Form fields for the opt-out-study action.
+ */
+export default class OptOutStudyFields extends ActionFields {
```

Comment:
This is just a comment because i'm not too familiar with this code.  What does ActionFields do?  The class seems to just have initialValues and doesn't inherit from redux-form.Field.